binaries/conda/build_packages.py

```diff
@@ -130,7 +130,7 @@ def conda_build(
 
     os.environ["PYTHON"] = "python"
 
-    python_versions = ["3.8", "3.9", "3.10"]
+    python_versions = ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
are you sure you wanna keep 3.8 around?

Reply:
We should support it as long as PyTorch does. But I think we should update our default version in workflows to 3.9